src/server.ts

```diff
@@ -3,7 +3,7 @@ import dotenv from "dotenv";
 import express from "express";
 import bodyParser from "body-parser";
 import axios from "axios";
-import jwt_decode from "jwt-decode";
+const bandwidthWebRTC = require("@bandwidth/webrtc");
```

Comment:
can we use the regular import statement here?

Reply:
We just merged a PR for that SDK to use TypeScript. I'll update it in a separate PR